internal/rbd/encryption.go

```diff
@@ -62,6 +62,9 @@ const (
 	metadataDEK    = "rbd.csi.ceph.com/dek"
 	oldMetadataDEK = ".rbd.csi.ceph.com/dek"
 
+	// luks2 header size metadata key.
+	luks2HeaderSizeKey = "rbd.csi.ceph.com/luks2HeaderSize"
```

Comment:
Do we want this metadata key to be present on all the clones/snapshots and also across cluster when mirroring is enabled?